hogehoge9.rb

```diff
@@ -0,0 +1,26 @@
+
+$id = 0
+
+puts "hello,world"
+
+
+if $id < 0
```

Comment:
Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.<br>Do not introduce global variables.